### ISSUE TITLE
Feature Request: Option to roll greed on non-upgrades

### ISSUE THREAD

**Issue by Jigain** [PunishXIV/LazyLoot#3]
Currently you can roll need on everything you can roll need on, and pass completely on items below a certain item level. I'm suggesting an option that if the item level of the item in question is significantly lower than your average item level, LazyLoot rolls greed for it rather than either needing or passing. Ideally this'd be expressed as a subtraction from average item level, my suggestion would be -30 but perhaps the user can input their own value.

In other words,
if ilvl < (average_ilvl - x) then greed, else need
